app/src/test/java/com/kickstarter/viewmodels/CommentsViewHolderViewModelTest.kt

```diff
@@ -598,20 +573,15 @@ class CommentsViewHolderViewModelTest : KSRobolectricTestCase() {
 
     @Test
     fun testCommentsViewModel_whenCommentModFFDisabled_shouldNotSetStatusToFlagged() {
```

Comment:
We should probably change the name of this test since it's not testing for the feature flag being enabled/disabled anymore.